packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/FLTCam.m

```diff
@@ -1199,6 +1199,64 @@ - (void)setZoomLevel:(CGFloat)zoom withCompletion:(void (^)(FlutterError *_Nulla
   completion(nil);
 }
 
+- (void)setVideoStabilizationMode:(FCPPlatformVideoStabilizationMode)mode
+                   withCompletion:(void (^)(FlutterError *_Nullable))completion {
+  AVCaptureVideoStabilizationMode avMode = getAvCaptureVideoStabilizationMode(mode);
+  bool isSupported = [self isVideoStabilizationAvModeSupported:avMode];
+
+  if (!isSupported) {
+    completion([FlutterError errorWithCode:@"VIDEO_STABILIIZATION_ERROR"
+                                   message:@"Unavailable video stabilization mode."
+                                   details:nil]);
+    return;
+  }
+
+  AVCaptureConnection *connection = [_captureVideoOutput connectionWithMediaType:AVMediaTypeVideo];
+
+  connection.preferredVideoStabilizationMode = avMode;
+
+  completion(nil);
+}
+
+- (bool)isVideoStabilizationModeSupported:(FCPPlatformVideoStabilizationMode)mode {
+  AVCaptureVideoStabilizationMode avMode = getAvCaptureVideoStabilizationMode(mode);
+
+  return [self isVideoStabilizationAvModeSupported:avMode];
+}
+
+- (bool)isVideoStabilizationAvModeSupported:(AVCaptureVideoStabilizationMode)avMode {
+  AVCaptureDeviceFormat *format = _captureDevice.activeFormat;
+  if (format == nil) {
+    return false;
+  }
+
+  bool isSupported = [format isVideoStabilizationModeSupported:avMode];
```

Comment:
you can directly pass the selector to `nil` to return NO

Reply:
Improved.